generic/tests/kdump.py

```diff
@@ -17,13 +17,17 @@ def preprocess_kdump(test, vm, timeout):
     """
     kdump_cfg_path = vm.params.get("kdump_cfg_path", "/etc/kdump.conf")
     auth_key_path = vm.params["auth_key_path"]
+    create_key_cmd = ("/bin/touch %s" % auth_key_path)
+    process.run(create_key_cmd, shell=True)
     backup_key_cmd = ("/bin/cp -f %s %s-bk" %
                       (auth_key_path, auth_key_path))
     cp_kdumpcf_cmd = "/bin/cp -f %s %s-bk" % (kdump_cfg_path, kdump_cfg_path)
     cp_kdumpcf_cmd = vm.params.get("cp_kdumpcf_cmd", cp_kdumpcf_cmd)
 
     session = vm.wait_for_login(timeout=timeout)
 
+    logging.info("Create authorized_keys file if it not existed.")
+    process.run(create_key_cmd, shell=True)
```

Comment:
I suppose the right approach here should be that skip backup and restore step when it does not exist.